akka-persistence-typed/src/main/scala/akka/persistence/typed/crdt/ORSet.scala

```diff
@@ -43,8 +43,7 @@ object ORSet {
     def underlying: ORSet[A]
   }
 
-  /** INTERNAL API */
-  @InternalApi private[akka] final case class AddDeltaOp[A](underlying: ORSet[A]) extends AtomicDeltaOp[A] {
+  final case class AddDeltaOp[A](underlying: ORSet[A]) extends AtomicDeltaOp[A] {
```

Comment:
Since it is case classes this means copy, apply, unapply, etc becomes public API/set in stone in addition to the field accessor. Might be fine but worth noting.

Reply:
Good point. I think we can remove the `case`. Could still be useful with `unapply` and toString`